node/impl/full/txhashmanager.go

```diff
@@ -84,13 +85,18 @@ func (m *EthTxHashManager) ProcessSignedMessage(ctx context.Context, msg *types.
 		return
 	}
 
-	ethTx, err := newEthTxFromSignedMessage(ctx, msg, m.StateAPI)
+	ethTx, err := ethtypes.EthTxFromSignedEthMessage(msg)
```

Comment:
Again, I'm just removing the state access (this is always a message from an EthAccount, so we never need to look at the state to resolve addresses).